common/src/main/java/com/wynntils/models/items/encoding/data/NameData.java

```diff
@@ -6,9 +6,23 @@
 
 import com.wynntils.models.items.encoding.type.ItemData;
 import com.wynntils.models.items.properties.IdentifiableItemProperty;
+import java.util.regex.Pattern;
 
 public record NameData(String name) implements ItemData {
+    private static final int MAX_NAME_LENGTH = 48;
+    private static final Pattern NAME_SANITIZER_PATTERN = Pattern.compile("[^a-zA-Z0-9 ]");
```

Comment:
I believe we also need to allow `'` and `-`, some default custom crafted names have this

Reply:
And I don't think we need 0-9, but not sure